tools/cldr-unittest/src/org/unicode/cldr/unittest/TestAnnotations.java

```diff
@@ -229,87 +212,61 @@ public void TestCategories() {
     }
 
     public void TestUniqueness() {
-//        if (logKnownIssue("cldrbug:10104", "Disable until the uniqueness problems are fixed")) {
-//            return;
-//        }
         Set<String> locales = new TreeSet<>();
-
         locales.add("en");
         locales.addAll(Annotations.getAvailable());
         locales.remove("root");
-//        if (getInclusion() < 6) {
-//            locales.retainAll(CLDRConfig.getInstance().getStandardCodes().getLocaleCoverageLocales(Organization.cldr));
-//        }
-        //locales.remove("sr_Latn");
-        Multimap<String, String> localeToNameToEmoji = TreeMultimap.create();
-        Multimap<String, String> nameToEmoji = TreeMultimap.create();
-        UnicodeMap<Annotations> english = Annotations.getData("en");
-        UnicodeSet englishKeys = getCurrent(english.keySet());
-        Map<String, UnicodeSet> localeToMissing = new TreeMap<>();
+        /*
+         * Note: "problems" here is a work-around for what appears to be a deficiency
+         * in the function sourceLocation, involving the call stack. Seemingly sourceLocation
+         * can't handle the "->" notation used for parallelStream().forEach() if
+         * uniquePerLocale calls errln directly.
+         */
+        Set<String> problems = new HashSet<String>();
+        locales.parallelStream().forEach(locale -> uniquePerLocale(locale, problems));
+        if (!problems.isEmpty()) {
+            problems.forEach(s -> errln(s));
+        }
+    }
```

Comment:
@btangmu blast from the past…do you know if there's a ticket for this issue?  Probably the -> lambda should just be skipped and find the enclosing stack frame. 

Looking at this test as part of CLDR-16947